packages/ui/src/services/color-config/plugin/create-color-config-plugin.ts

```diff
@@ -1,17 +1,16 @@
+import { PartialGlobalConfig } from './../../global-config/types'
 import { ColorVariables } from './../types'
 import { App, watch } from 'vue'
 import { isServer } from '../../../utils/ssr-utils'
-import { getGlobalProperty } from '../../../vuestic-plugin/utils'
 import { cssVariableName } from '../utils'
-import { getTextColor } from '../color-functions'
 import { useColors } from '../../../composables'
 
 export const setCSSVariable = (name: string, value: string, root: HTMLElement) => {
   root.style.setProperty(cssVariableName(name), value)
 }
 
-export const createColorConfigPlugin = (app: App) => {
-  const { colors, getTextColor, getColor } = useColors()
+export const createColorConfigPlugin = (app: App, config: PartialGlobalConfig | undefined) => {
```

Comment:
Can it be replaced like this? Or it will be tough to read it?
```suggestion
export const createColorConfigPlugin = (app: App, config?: PartialGlobalConfig) => {
```